rbac/management/group/view.py

```diff
@@ -250,7 +251,12 @@ def create(self, request, *args, **kwargs):
             }
         """
         validate_group_name(request.data.get("name"))
-        return super().create(request=request, args=args, kwargs=kwargs)
+        create_group = super().create(request=request, args=args, kwargs=kwargs)
+
+        if status.is_success(create_group.status_code):
+            auditlog = AuditLog()
+            auditlog.log_create(request, AuditLog.GROUP)
+            return create_group
```

Comment:
I think this should be moved out of the If statement

Reply:
it looks this needs to be fixed also for roles: https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/role/view.py#L218

Reply:
However, it might not be necessary since it only fails in cases where it doesn't return the entire object.